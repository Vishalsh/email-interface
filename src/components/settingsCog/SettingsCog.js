import React from 'react';

import classes from './SettingsCog.module.scss';

const SettingsCog = () => {
  return (
    <div className={classes.settingsCog}>
      <i className="icon icon-settings"/>
    </div>
  );
};

export default SettingsCog;
