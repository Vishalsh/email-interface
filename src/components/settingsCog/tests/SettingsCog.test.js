import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import SettingsCog from '../SettingsCog';

describe('SettingsCog', () => {
  it('should render the component', () => {
    const component = shallow(<SettingsCog />);

    expect(toJson(component)).toMatchSnapshot();
  });
});