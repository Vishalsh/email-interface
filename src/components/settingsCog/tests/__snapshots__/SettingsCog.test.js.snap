// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SettingsCog should render the component 1`] = `
<div
  className="settingsCog"
>
  <i
    className="icon icon-settings"
  />
</div>
`;
